### PR TITLE
feat: add type module [test] [ignore]

### PR DESCRIPTION
**Pull request by saurabhdaware**
This makes things work on Vite

### CODE REVIEW

File: packages/blade/package.json [hunk 1]
```diff
@@ -1,11 +1,12 @@
 {
   "name": "@razorpay/blade",
   "description": "The Design System that powers Razorpay",
-  "version": "7.1.0",
+  "version": "7.1.0-module-test.1",
   "license": "MIT",
   "engines": {
     "node": ">=14.18.0"
   },
+  "type": "module",
```
Comment (kamleshchandnani) [inline packages/blade/package.json:9]: why was this needed? 🤔

**Comment by saurabhdaware** [comment]
Will re-open and figure out later
